profiler/src/ProfilerEngine/Datadog.Profiler.Native/OsSpecificApi.h

```diff
@@ -23,4 +23,5 @@ namespace OsSpecificApi
    bool IsRunning(IThreadInfo* pThreadInfo, uint64_t& cpuTime, bool& failed);
    int32_t GetProcessorCount();
    std::vector<std::shared_ptr<IThreadInfo>> GetProcessThreads();
+   bool GetLastErrorMessage(DWORD& errorCode, std::string& message);
```

Comment:
Why returning `bool` and not using it ?
Why not returning `std::tuple<DWORD, std::string>` instead ?

Reply:
✅ done